internal/configs/virtualserver.go

```diff
@@ -676,7 +685,8 @@ func (vsc *virtualServerConfigurator) GenerateVirtualServerConfig(
 			VSName:                    vsEx.VirtualServer.Name,
 			DisableIPV6:               vsc.isIPV6Disabled,
 		},
-		SpiffeCerts: vsc.spiffeCerts,
+		SpiffeCerts:       enabledInternalRoutes,
+		SpiffeClientCerts: vsc.spiffeCerts,
```

Comment:
does this need to be similar to what we have for Ingress?
```
SpiffeClientCerts: staticParams.NginxServiceMesh && !cfgParams.SpiffeServerCerts
```

Reply:
Yes, that is necessary to prevent customer misconfiguration. I just pushed a fix.

Reply:
Thanks for catching that!